ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -335,18 +335,39 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
         
         progressView.progress = Float(quiz.currentIndex) / Float(questions.count)
         progressView.set(minY: progressView.frame.minY + 10)
-        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: 3.0))
+        
+        let progressViewHeight: CGFloat = 6
+        // This is according to Stack Overflow the best way to give `UIProgressView` a custom height.
+        // we're not hard-coding the denominator here, as in iOS 14 Apple changed the standard height.
+        // frame may be set to 22 above, but something in `UIProgressView`'s init method, manually sets
+        // it to the correct system default value!
+        let yTransform = progressViewHeight/progressView.bounds.height
+        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: yTransform))
         
         progressContainer.addSubview(progressView)
         
-        let progressStartCap = UIView(frame: CGRect(x: progressView.frame.minX - 2, y: progressView.frame.minY, width: 6, height: 6))
-        progressStartCap.layer.cornerRadius = 3.0
+        let progressStartCap = UIView(
+            frame: CGRect(
+                x: progressView.frame.minX - 2,
+                y: progressView.frame.minY,
+                width: progressViewHeight,
+                height: progressViewHeight
+            )
+        )
+        progressStartCap.layer.cornerRadius = progressViewHeight/2
         progressStartCap.layer.masksToBounds = true
         progressStartCap.backgroundColor = progressView.progressTintColor
         progressContainer.addSubview(progressStartCap)
         
-        let progressEndCap = UIView(frame: CGRect(x: progressView.frame.maxX - 3, y: progressView.frame.minY, width: 6, height: 6))
-        progressEndCap.layer.cornerRadius = 3.0
+        let progressEndCap = UIView(
+            frame: CGRect(
+                x: progressView.frame.maxX - 4,
```

Comment:
I know you've only cleaned up formatting but weird the min goes to the left and the max ALSO goes to the left by a different amount? Symmetry feels weird here